docs/faq.rst

```diff
@@ -35,7 +35,7 @@ actually work. The second reason, which is still valid today, is that
 ``activate.fish`` tries to modify your ``fish_prompt`` function.
 
 Because ``fish_prompt`` is a function and not a variable like in most other
-shells, modifying it programatically is not trivial, and the way that
+shells, modifying it programmatically is not trivial, and the way that
 virtualenv accomplishes it is more than a little hacky. The benefit of it being
 a function is that the syntax for customising it is much less terse and cryptic
```

Comment:
I left *customizing* intact, I believe it's the Australian spelling of customize.